### PR TITLE
Glance role using openstack-config

### PR DESCRIPTION
**Pull request by adean20csc**
Glance role using openstack-config

### CODE REVIEW

File: group_vars/all [hunk 1]
```diff
@@ -87,7 +87,9 @@ controller_nodes:
 
 
 ### Messaging
-rabbit_host: 172.17.17.70
```
Comment (mtnbikenc) [inline group_vars/all:90]: Removing rabbit_host breaks a lot of other roles that use that var.

File: hosts [hunk 2]
```diff
@@ -25,6 +26,8 @@ wiley-ctlr-3 ansible_ssh_host=172.17.16.72
 [osp:children]
```
Comment (mtnbikenc) [inline hosts:26]: Doesn't appear that the host group osp is used in a playbook anywhere.  Why are we adding host groups to this group?  And group swift does not exist.

**Comment by jdextercsc** [comment]
We want to use rabbit_hosts IP:port,ip:port....  instead of rabbit_host which is a single entry.

**Comment by jdextercsc** [comment]
group swift will be added soon

File: roles/glance/tasks/main.yml [hunk 3]
```diff
@@ -21,27 +75,17 @@
   when: ansible_distribution_major_version|int == 7
   tags: glance
 
+-name: use pacemaker to mount the NFS share as service.
+  shell: pcs resource create glance-fs Filesystem device="{{ wiley-ctlr-1 }}:{{ glance_datadir }}" directory="{{ glance_datadir }}" fstype="nfs" options="v3" --clone
+  register: cmd
+  changed_when: “‘unchanged’ not in cmd.stderr"
```
Comment (mtnbikenc) [inline roles/glance/tasks/main.yml]: Not sure this stderr works the same as it does with openstack-config.

**Comment by mtnbikenc** [comment]
Can you add the j2 templates back in unmodified?

**Comment by mtnbikenc** [comment]
Code review :+1:

**Pull request merged**